src/types/reduxActions.js

```diff
@@ -178,12 +166,57 @@ export type Action =
     }
   | { type: 'CONTACTS/LOAD_CONTACTS_SUCCESS', data: { contacts: GuiContact[] } }
   | { type: 'GENERIC_SHAPE_SHIFT_ERROR', data: string }
+  | { type: 'NEW_RECEIVE_ADDRESS', data: { receiveAddress: EdgeReceiveAddress } }
   | { type: 'PARSE_URI_SUCCEEDED', data: { parsedUri: EdgeParsedUri } }
+  | { type: 'RESET_WALLET_LOADING_PROGRESS', data: { walletId: string } }
   | { type: 'SAVE_EDGE_LOBBY', data: EdgeLobby }
   | { type: 'SET_LOBBY_ERROR', data: string }
   | { type: 'SET_FROM_WALLET_MAX', data: string }
+  | { type: 'SET_TRANSACTION_SUBCATEGORIES', data: { subcategories: string[] } }
+  | { type: 'SPENDING_LIMITS/NEW_SPENDING_LIMITS', data: { spendingLimits: SpendingLimits } }
+  | {
+      type: 'UI/SCENES/TRANSACTION_LIST/UPDATE_TRANSACTIONS',
+      data: {
+        numTransactions: number,
+        transactions: TransactionListTx[],
+        transactionIdMap: { [txid: string]: TransactionListTx },
+        currentCurrencyCode: string,
+        currentWalletId: string,
+        currentEndIndex: number
+      }
+    }
+  | { type: 'UI/SEND_CONFIMATION/NEW_PIN', data: { pin: string } }
```

Comment:
just a proposal, we could change `SEND_CONFIMATION` to `SEND_CONFIRMATION`

Reply:
Good idea, but I'll do it in a separate commit.